evap/results/exporters.py

```diff
@@ -143,7 +143,7 @@ def export(self, response, course_types_list, include_not_enough_voters=False, i
                                     total_count += grade_result.total_count
                                     if grade_result.question.is_yes_no_question:
                                         approval_count += grade_result.approval_count
-                        if values and course.has_enough_voters_to_publish_grades:
+                        if values and course.has_enough_voters_to_publish_grade:
```

Comment:
we're checking whether the average grade can be published and only then `avg` is computed and added to the export. otherwise we include an empty cell.

Reply:
this code here is about individual questions, not the total average grade.

Reply:
you're right, thanks